src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/CaseAllocationService.kt

```diff
@@ -106,4 +109,11 @@ class CaseAllocationService(
     }
     return caseAllocationEntity?.let { caseAllocationRepository.save(it) }
   }
+
+  @Transactional
+  fun getAllResettlementWorkers(
+    prisonId: String,
+  ): List<ManageUser> {
+    return manageUserApiService.getManageUsersData(prisonId, "ADD_SENSITIVE_CASE_NOTES")
```

Comment:
Should be `PSFR_RESETTLEMENT_WORKER` here

Reply:
done